### PR TITLE
chore: Read service/git provider

### PR DESCRIPTION
**Pull request by Shivam-nagar23**
# Description

Fixes #

## Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit/api test cases.

## Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/build/git/gitHost/read/gitHostReadService.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	// get orchestrator host
+	orchestratorHost, err := impl.attributeService.GetByKey("url")
+	if err != nil {
+		impl.logger.Errorw("error in fetching orchestrator host url from db", "err", err)
+		return nil, err
+	}
+
+	var webhookUrlPrepend string
+	if orchestratorHost == nil || len(orchestratorHost.Value) == 0 {
+		webhookUrlPrepend = "{HOST_URL_PLACEHOLDER}"
```
Comment (kartik-579) [inline pkg/build/git/gitHost/read/gitHostReadService.go:93]: constant

File: pkg/build/git/gitProvider/read/gitProviderReadService.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+			EnableTLSVerification: provider.EnableTLSVerification,
+			TLSConfig: bean.TLSConfig{
+				CaData:      "",
+				TLSCertData: "",
+				TLSKeyData:  "",
+			},
+			IsCADataPresent:      len(provider.CaCert) > 0,
+			IsTLSKeyDataPresent:  len(provider.TlsKey) > 0,
+			IsTLSCertDataPresent: len(provider.TlsCert) > 0,
+		}
```
Comment (kartik-579) [inline pkg/build/git/gitProvider/read/gitProviderReadService.go:91]: move to adapter

**Review by kartik-579 — CHANGES_REQUESTED** [review]

File: pkg/build/git/gitHost/read/gitHostReadService.go (hunk 1 above)
Comment (Shivam-nagar23) [reply pkg/build/git/gitHost/read/gitHostReadService.go:93]: done

File: pkg/build/git/gitProvider/read/gitProviderReadService.go (hunk 2 above)
Comment (Shivam-nagar23) [reply pkg/build/git/gitProvider/read/gitProviderReadService.go]: done

**Review by kartik-579 — APPROVED** [review]

**Review by kartik-579 — APPROVED** [review]

**Review by vikramdevtron — APPROVED** [review]

**Pull request merged**